packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -373,6 +373,10 @@ export async function getResourceGroupNameFromEnv(env: string): Promise<string |
 }
 
 export async function getProvisionSucceedFromEnv(env: string): Promise<boolean | undefined> {
+  if (isV3Enabled()) {
```

Comment:
I know it is hard to detect whether provisioned actually in V3. This will make all environment nodes except local always show `(Provisioned)` description like in GIF. Is this UI expected?

Reply:
Sorry, it is only my local test, committed this change by mistake, let me revert it.

Reply:
No problem, I'll change this part later.